src/test/java/bob/DeadlineTest.java

```diff
@@ -0,0 +1,22 @@
+package bob;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.time.LocalDate;
+
+public class DeadlineTest {
+    @Test
+    public void testOne() {
```

Comment:
Perhaps you can make the names of the test functions more descriptive?

Reply:
Thanks for the suggestiong! Will edit the test names!